docs/modules/ROOT/pages/index.adoc

```diff
@@ -17,21 +17,21 @@ for tables having Change Data Capture (CDC) enabled to https://www.datastax.com/
 {cdc_cass} has two components:
 
 * DataStax Change Agent for Apache Cassandra, which is an event producer deployed as a JVM agent on each Cassandra data node
-* {csc_pulsar_first}, which is a source connector deployed in your streaming platform
+* {cdc_pulsar}, which is a source connector deployed in your streaming platform
 
 The following diagram describes the general architecture.
 
 image::cdc-for-cassandra-overview.png[]
 
 Since version 3.0, Cassandra has included a change data capture (CDC) feature.
-The CDC feature can be enabled on the table level by setting the table property cdc=true, after which any commit log containing data for a CDC-enabled table will be moved to the CDC directory specified in `cassandra.yaml` on discard/flush (default: `cdc_raw`).
+The CDC feature can be enabled on the table level by setting the table property `cdc=true`, after which any commit log containing data for a CDC-enabled table will be moved to the CDC directory specified in `cassandra.yaml` on discard/flush (default: `cdc_raw`).
 Since Cassandra 4.0, commit logs can be synced to the `cdc_raw` directory periodically (by default, 10 seconds).
```

Comment:
Since Cassandra 4.0 (and back ported in C* 3.11+), C* CDC feature has been improved such that commit logs can ... . This improvement makes C* CDC feature near real time.